forte/data/extractors/seqtagging_extractor.py

```diff
@@ -199,11 +202,15 @@ def extract(self, pack: DataPack, instance: Annotation) -> Feature:
             raw_data: List = vocab_mapped
             if self.is_bert:
                 raw_data = [pad_value] + raw_data + [pad_value]
+
+            need_pad = self.vocab.use_pad
         else:
             # When vocabulary is not available, use the original data.
             raw_data = instance_tagged
+            need_pad = self.config.need_pad
 
-        meta_data = {"pad_value": pad_value,
+        meta_data = {"need_pad": need_pad,
```

Comment:
can we always use `self.vocab.use_pad`? Because if `self.config.need_pad` will be passed to `self.vocab.use_pad`?

Reply:
self.vocab is Optional[Vocabulary], so if will throw mypy error if I set it to self.vocab.use_pad.

Reply:
we can check if vocab exsits and throw exceptions first, and then use `self.vocab.use_pad`